### PR TITLE
feat(cdk)!: rename `TuiAbstractTextfieldHost` to `AbstractTuiTextfieldHost`

### LINKED ISSUE #2485: 🤖 - Inspect `TuiAbstractTextfieldHost`
### Description

We can probably get rid of `process` method and rely on `nativeFocusableElement` from the host. This issue is here so we don't forget to take a look in 3.0

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #2485

### CODE REVIEW

File: projects/core/abstract/driver.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import {ExistingProvider, Type} from '@angular/core';
 import {Observable} from 'rxjs';
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
 export abstract class TuiDriver extends Observable<boolean> {}
```
Comment (splincode) [inline projects/core/abstract/driver.ts:5]: @waterplea do we need rename to `AbstractTuiDriver`?

File: projects/core/abstract/position-accessor.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import {ExistingProvider, Type} from '@angular/core';
 import {TuiPoint} from '@taiga-ui/core/types';
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
 export abstract class TuiPositionAccessor {
```
Comment (splincode) [inline projects/core/abstract/position-accessor.ts:5]: @waterplea do we need rename to `AbstractTuiPositionAccessor`?

File: projects/core/abstract/rect-accessor.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import {ExistingProvider, Type} from '@angular/core';
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
 export abstract class TuiRectAccessor {
```
Comment (splincode) [inline projects/core/abstract/rect-accessor.ts:4]: @waterplea do we need rename to `AbstractTuiRectAccessor`?

File: projects/core/abstract/vehicle.ts [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import {ExistingProvider, Type} from '@angular/core';
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
 export abstract class TuiVehicle {
```
Comment (splincode) [inline projects/core/abstract/vehicle.ts:4]: @waterplea do we need rename to `AbstractTuiVehicle`?

File: projects/core/abstract/driver.ts (hunk 1 above)
Comment (waterplea) [reply projects/core/abstract/driver.ts:5]: I don't think so. It's kind of like Angular has some abstract classes that don't look like abstract in their name. I feel it is more natural this way for those.

**Review by waterplea — APPROVED** [review]

File: projects/cdk/schematics/ng-update/tests/schematic-rename-types.spec.ts [hunk 5]
```diff
@@ -72,14 +73,14 @@ export class AppComponent {
    }
    constructor(@Inject(TUI_INPUT_TIME_OPTIONS) readonly options: InputTimeOptions) {}
 }
+
+export class Textfield extends TuiAbstractTextfieldHost {}
 `;
 
+// @ts-ignore
```
Comment (nsbarsukov) [inline projects/cdk/schematics/ng-update/tests/schematic-rename-types.spec.ts:80]: Why do we need it ?

**Review by vladimirpotekhin — APPROVED** [review]

**Review by nsbarsukov — APPROVED** [review]

File: projects/cdk/schematics/ng-update/tests/schematic-rename-types.spec.ts (hunk 5 above)
Comment (splincode) [reply projects/cdk/schematics/ng-update/tests/schematic-rename-types.spec.ts]: fixed

**Pull request merged**